geoprocessing/src/modules/planning-units/planning-units.geo.entity.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @todo move the planningUnitsGeom entity to the api
+ */
+import { ApiProperty } from '@nestjs/swagger';
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+@Entity('planning_units_geom')
+export class PlanningUnitsGeom {
+  /**
+   * id
+   */
+  @ApiProperty()
+  @PrimaryColumn('uuid', { name: 'id' })
+  id: string;
+
+  /**
+   * geometry column.
+   */
+  @ApiProperty()
+  @Column('geometry', { name: 'the_geom', select: false })
+  theGeom: any;
```

Comment:
We should avoid using `any` whenever possible. 

Take a look [there](https://github.com/Vizzuality/marxan-cloud/pull/150/files#diff-e860777d8791413043fe2da6becf6ed22d1471e696f4f5a13bd78460f1d69f2eR46) for proper typing on `Geometry`